manifest.yaml

```diff
@@ -52,7 +52,8 @@ arguments:
           - grpc
           - temporal
           - kafka
-    description: Any valid built-in service activities can be supplied here and they will be provided. The `service` flag must be set to true. Valid values for this are `http`, `grpc`, `temporal`, and `kafka`.
+          - python
+    description: Any valid built-in service activities can be supplied here and they will be provided. The `service` flag must be set to true. Valid values for this are `http`, `grpc`, `temporal`, `python` and `kafka`.
```

Comment:
Same comment here as the stencil-golang PR. Also, if this is somehow causing problems, we should remove the schema from this field. 

We're going to be removing any mentions/usage of `serviceActivites` from this module because it's meant to be apart of stencil-golang only.

Reply:
I'm adding here just for consistency purpose. If we are going to delete this in the future, I'm fine to keep it as is.

Reply:
found you actually can't delete the definition easily from here since it's being used in Readme, and I need to modify the Readme to make python developer to use the right command too (a wrapper to support python image build).

Reply:
It's not blocking your work. I went ahead and just did it in https://github.com/getoutreach/stencil-base/pull/108.